packages/tasks/src/model-libraries.ts

```diff
@@ -1022,6 +1022,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://github.com/jasonppy/VoiceCraft",
 		snippets: snippets.voicecraft,
 	},
+	vui: {
+		prettyLabel: "VUI",
+		repoName: "VUI",
```

Comment:
```suggestion
		prettyLabel: "Vui",
		repoName: "Vui",
```

(taken from their modelcard:

> Vui.BASE is base checkpoint trained on 40k hours of audio conversations Vui.ABRAHAM is a single speaker model that can reply with context awareness. Vui.COHOST is checkpoint with two speakers that can talk to each other.

)